{"ygPerms":{"resourceCapabilityList":[{"resourceType":"GROUP","capabilities":[{"name":"READ"},{"name":"JOIN"}]},{"resourceType":"PHOTO","capabilities":[]},{"resourceType":"FILE","capabilities":[]},{"resourceType":"MEMBER","capabilities":[]},{"resourceType":"LINK","capabilities":[]},{"resourceType":"CALENDAR","capabilities":[]},{"resourceType":"DATABASE","capabilities":[]},{"resourceType":"POLL","capabilities":[]},{"resourceType":"MESSAGE","capabilities":[{"name":"READ"}]},{"resourceType":"PENDING_MESSAGE","capabilities":[]},{"resourceType":"ATTACHMENTS","capabilities":[{"name":"READ"}]},{"resourceType":"PHOTOMATIC_ALBUMS","capabilities":[]},{"resourceType":"MEMBERSHIP_TYPE","capabilities":[]},{"resourceType":"POST","capabilities":[{"name":"READ"}]},{"resourceType":"PIN","capabilities":[]}],"groupUrl":"groups.yahoo.com","intlCode":"us"},"comscore":"pageview_candidate","ygData":{"userId":44728387,"authorName":"Dave Goodwin","from":"Dave Goodwin &lt;dgoodwin@...&gt;","profile":"dggoodwin","replyTo":"LIST","senderId":"WXnXPA3vd636IrQNqIVCo3Q0ZnFH8vE0k7FhatISrlMqJOiMqgpZndsrFbeZgvEy-db3jZqypwzdKyal2YzqMrcxztpq79DFZdY","spamInfo":{"isSpam":false,"reason":"12"},"subject":"Re: [cantera] ImportError: No module named ctml_writer, OS X","postDate":"1196571845","msgId":1777,"canDelete":false,"contentTrasformed":false,"systemMessage":false,"headers":{"messageIdInHeader":"PEQ3RDA1MkM3LTU5NDUtNDBGNC04QjRDLTI2RDdGQ0E3NEY1Q0BjYWx0ZWNoLmVkdT4=","inReplyToHeader":"PDBEQUNBREZBLUEwMUEtNEQwMS04NDczLTA3MUFERTNCMDUxOUB1YWYuZWR1Pg==","referencesHeader":"PDBEQUNBREZBLUEwMUEtNEQwMS04NDczLTA3MUFERTNCMDUxOUB1YWYuZWR1Pg=="},"prevInTopic":1773,"nextInTopic":1778,"prevInTime":1776,"nextInTime":1778,"topicId":1773,"numMessagesInTopic":7,"msgSnippet":"It looks to me like you have more than one Python installation. Be sure to run the script setup_cantera to set environment variable PYTHON_CMD to the Python","rawEmail":"Return-Path: &lt;dgoodwin@...&gt;\r\nX-Sender: dgoodwin@...\r\nX-Apparently-To: cantera@yahoogroups.com\r\nX-Received: (qmail 88570 invoked from network); 2 Dec 2007 05:04:22 -0000\r\nX-Received: from unknown (66.218.67.96)\n  by m56.grp.scd.yahoo.com with QMQP; 2 Dec 2007 05:04:22 -0000\r\nX-Received: from unknown (HELO outgoing-mail.its.caltech.edu) (131.215.239.19)\n  by mta17.grp.scd.yahoo.com with SMTP; 2 Dec 2007 05:04:22 -0000\r\nX-Received: from earth-dog.its.caltech.edu (earth-dog [192.168.1.3])\n\tby water-ox-postvirus (Postfix) with ESMTP id A87181BAE3\n\tfor &lt;cantera@yahoogroups.com&gt;; Sat,  1 Dec 2007 21:04:09 -0800 (PST)\r\nX-Received: from [10.0.1.6] (68-190-217-240.dhcp.psdn.ca.charter.com [68.190.217.240])\n\t(Authenticated sender: dgg)\n\tby water-ox.its.caltech.edu (Postfix) with ESMTP id 8D7D61BAE6\n\tfor &lt;cantera@yahoogroups.com&gt;; Sat,  1 Dec 2007 21:04:04 -0800 (PST)\r\nMime-Version: 1.0 (Apple Message framework v752.3)\r\nIn-Reply-To: &lt;0DACADFA-A01A-4D01-8473-071ADE3B0519@...&gt;\r\nReferences: &lt;0DACADFA-A01A-4D01-8473-071ADE3B0519@...&gt;\r\nContent-Type: text/plain; charset=US-ASCII; delsp=yes; format=flowed\r\nMessage-Id: &lt;D7D052C7-5945-40F4-8B4C-26D7FCA74F5C@...&gt;\r\nContent-Transfer-Encoding: 7bit\r\nDate: Sat, 1 Dec 2007 21:04:05 -0800\r\nTo: cantera@yahoogroups.com\r\nX-Mailer: Apple Mail (2.752.3)\r\nX-Spam-Scanned: at Caltech-ITS on earth-dog by amavisd-2.4.5\r\nX-eGroups-Msg-Info: 1:12:0:0:0\r\nFrom: Dave Goodwin &lt;dgoodwin@...&gt;\r\nSubject: Re: [cantera] ImportError: No module named ctml_writer, OS X\r\nX-Yahoo-Group-Post: member; u=44728387; y=F8DRgqHeWZmODZI-JH0i3nEPlrOod5F9jfaxmLGkC8Bk\r\nX-Yahoo-Profile: dggoodwin\r\n\r\nIt looks to me like you have more than one Python installation. Be  \nsure to run the script setup_cantera to set environment variable  \nPYTHON_CMD to the Python interpreter used to install Cantera. If you  \ninstalled a version of Python yourself, but the system Python  \nintrpreter in /usr/bin comes first in PATH, this sort of problem will  \noccur.\n\n\nOn Nov 28, 2007, at 10:12 AM, Winston Burbank wrote:\n\n&gt; I am running Cantera from Matlab on an OS X 10.5 machine.  Since\n&gt; upgrading to 10.5 Cantera has worked just fine, however I once got\n&gt; frustrated with the X11 Simlink interface and rebooted into Windows XP\n&gt; on the same machine and manipulated the same Matlab files that I use\n&gt; from OS X.  These are all things that have done before without error.\n&gt; However, rebooting into OS X and starting Matlab I now get the\n&gt; following errors.\n&gt;\n&gt; I have tried (without uninstalling) reinstalling everything but Matlab\n&gt; (python, numarray, Sundials, Cantera).  Running the adiabatic.py demo\n&gt; also returns an error (see the end for the error).  I have Python 2.5\n&gt; installed and my Library/Python/2.5/site-packages folder is empty??\n&gt;\n&gt; Please help as I have no idea what to try next, and thank you for your\n&gt; time and effort for my cause,\n&gt; Winston Burbank\n&gt;\n&gt; *From my original folder with the XML files (still works under XP)\n&gt;\n&gt;&gt;&gt; run(&#39;/Volumes/DATA/Matlab Work/Cantera_Test_Wolf_Solu/assump.m&#39;)\n&gt;   Traceback (most recent call last):\n&gt;     File &quot;./.cttmp1196269629.pyw&quot;, line 1, in &lt;module&gt;\n&gt;       from ctml_writer import *\n&gt;   ImportError: No module named ctml_writer\n&gt;\n&gt; ??? Error using ==&gt; run at 56\n&gt; Error using ==&gt; run at 69\n&gt; Error using ==&gt; ThermoPhase.ThermoPhase at 14\n&gt;\n&gt;\n&gt; ************************************************\n&gt;                  Cantera Error!\n&gt; ************************************************\n&gt;\n&gt;\n&gt; Procedure: importPhase\n&gt; Error:   no data for species\n&gt; ______________________________________________________________________ \n&gt; ___________\n&gt;\n&gt;\n&gt; * I then thought maybe the XML files where different as I am using the\n&gt; stable release of Cantera on XP and the cvs version on OS X.  So I\n&gt; copied the folder and trashed the XML files expecting Cantera to\n&gt; recreate them, however I get the following error with the XML files\n&gt; never being created.\n&gt;\n&gt;&gt;&gt; run(&#39;/Volumes/DATA/Matlab Work/PHD Thesis/assump.m&#39;)\n&gt;   Traceback (most recent call last):\n&gt;     File &quot;./.cttmp1196269322.pyw&quot;, line 1, in &lt;module&gt;\n&gt;       from ctml_writer import *\n&gt;   ImportError: No module named ctml_writer\n&gt;\n&gt; ??? Error using ==&gt; run at 56\n&gt; Error using ==&gt; run at 69\n&gt;\n&gt;\n&gt; ************************************************\n&gt;                  Cantera Error!\n&gt; ************************************************\n&gt;\n&gt;\n&gt; Procedure: get_XML_File\n&gt; Error:   cannot open ./Basic.xml for reading.Note, this error\n&gt; indicates a possible configuration problem.\n&gt; ______________________________________________________________________ \n&gt; ___________\n&gt;\n&gt; To make the error even more clear I have run the following from  \n&gt; Matlab:\n&gt;\n&gt;&gt;&gt; gas= importPhase(&#39;gri30.cti&#39;);\n&gt; ??? Error using ==&gt; ThermoPhase.ThermoPhase at 14\n&gt;\n&gt;\n&gt; ************************************************\n&gt;                  Cantera Error!\n&gt; ************************************************\n&gt;\n&gt;\n&gt; Procedure: importPhase\n&gt; Error:   no data for species\n&gt;\n&gt;\n&gt; Error in ==&gt; Solution.Solution at 36\n&gt; t = ThermoPhase(node);\n&gt;\n&gt; Error in ==&gt; importPhase at 5\n&gt;    s = Solution(file);\n&gt; ______________________________________________________________________ \n&gt; ___________\n&gt;\n&gt;\n&gt;&gt;&gt; gas= importPhase(&#39;gri30.xml&#39;);\n&gt; ??? Error using ==&gt; ThermoPhase.ThermoPhase at 14\n&gt;\n&gt;\n&gt; ************************************************\n&gt;                  Cantera Error!\n&gt; ************************************************\n&gt;\n&gt;\n&gt; Procedure: importPhase\n&gt; Error:   no data for species\n&gt;\n&gt;\n&gt; Error in ==&gt; Solution.Solution at 36\n&gt; t = ThermoPhase(node);\n&gt;\n&gt; Error in ==&gt; importPhase at 5\n&gt;    s = Solution(file);\n&gt; ______________________________________________________________________ \n&gt; ___________\n&gt;\n&gt; Python adiabatic.py error:\n&gt;\n&gt; Traceback (most recent call last):\n&gt;    File &quot;/Applications/cantera/demos/python/equilibrium/adiabatic.py&quot;,\n&gt; line 21, in &lt;module&gt;\n&gt;      gas  = importPhase(&#39;gri30.cti&#39;)\n&gt;    File &quot;/Library/Frameworks/Python.framework/Versions/2.5/lib/\n&gt; python2.5/site-packages/Cantera/importFromFile.py&quot;, line 20, in\n&gt; importPhase\n&gt;      return importPhases(file, [name], loglevel, debug)[0]\n&gt;    File &quot;/Library/Frameworks/Python.framework/Versions/2.5/lib/\n&gt; python2.5/site-packages/Cantera/importFromFile.py&quot;, line 27, in\n&gt; importPhases\n&gt;\n&gt; s\n&gt; .append(solution.Solution \n&gt; (src=file,id=nm,loglevel=loglevel,debug=debug))\n&gt;    File &quot;/Library/Frameworks/Python.framework/Versions/2.5/lib/\n&gt; python2.5/site-packages/Cantera/solution.py&quot;, line 39, in __init__\n&gt;      preprocess = 1, debug = debug)\n&gt;    File &quot;/Library/Frameworks/Python.framework/Versions/2.5/lib/\n&gt; python2.5/site-packages/Cantera/XML.py&quot;, line 35, in __init__\n&gt;      self._xml_id = _cantera.xml_get_XML_File(src, debug)\n&gt; error:\n&gt;\n&gt; ************************************************\n&gt;                  Cantera Error!\n&gt; ************************************************\n&gt;\n&gt;\n&gt; Procedure: ct2ctml\n&gt; Error:   cannot open ./.cttmp1196271005.pyw for writing.\n&gt;\n&gt;\n&gt;\n&gt; Yahoo! Groups Links\n&gt;\n&gt;\n&gt;\n\n\n"}}